concourse/pipelines/deploy.yml

```diff
@@ -447,44 +458,54 @@ jobs:
     plan:
     - in_parallel:
       - get: govuk-infrastructure
-        resource: govuk-infrastructure-publisher
+        resource: govuk-infrastructure-concourse-tasks
         trigger: true
-      - get: govuk-terraform-outputs
+      - get: app-terraform-outputs
+        resource: publisher-terraform-outputs
         passed:
         - run-terraform
         trigger: true
       - get: release
         resource: publisher
         trigger: true
-    - task: update-web-task-definition
-      file: govuk-infrastructure/concourse/tasks/update-task-definition.yml
-      params:
-        APPLICATION: publisher-web
-        GOVUK_ENVIRONMENT: test
-    - task: update-worker-task-definition
-      file: govuk-infrastructure/concourse/tasks/update-task-definition.yml
-      params:
-        APPLICATION: publisher-worker
-        GOVUK_ENVIRONMENT: test
+    - in_parallel:
+      - task: update-web-task-definition
+        file: govuk-infrastructure/concourse/tasks/update-task-definition-v2.yml
+        output_mapping:
+          task-definition-arn: web-task-definition-arn
+        params:
+          APPLICATION: publisher
+          VARIANT: web
+          GOVUK_ENVIRONMENT: test
+      - task: update-worker-task-definition
+        file: govuk-infrastructure/concourse/tasks/update-task-definition-v2.yml
+        output_mapping:
+          task-definition-arn: worker-task-definition-arn
+        params:
+          APPLICATION: publisher
+          VARIANT: worker
+          GOVUK_ENVIRONMENT: test
     - task: run-db-migrations
       file: govuk-infrastructure/concourse/tasks/run-task.yml
+      input_mapping:
+        task-definition-arn: web-task-definition-arn
       params:
-        APPLICATION: publisher-web
+        APPLICATION: publisher
         COMMAND: "bundle exec rails db:migrate"
-        TASK_DEFINITION: web_task_definition
+        TASK_DEFINITION: web-task-definition-arn
```

Comment:
I don't think you need this anymore, now that you've got the `input_mapping`. It doesn't look like run-task.yml refers to this anymore.